src/Poseidon/Package.hs

```diff
@@ -247,6 +251,28 @@ readPoseidonPackageCollection opts dirs = do
     -- return package list
     return finalPackageList
   where
+    filterByPoseidonVersion :: [FilePath] -> IO [FilePath]
+    filterByPoseidonVersion posFiles = do
+        eitherPaths <- mapM isInVersionRange posFiles
+        mapM_ (hPutStrLn stderr . renderPoseidonException) $ lefts eitherPaths
+        return $ rights eitherPaths
+        where 
+            isInVersionRange :: FilePath -> IO (Either PoseidonException FilePath)
+            isInVersionRange posFile = do
+                content <- readFile' posFile
+                let posLines = lines content
+                case elemIndex "poseidonVersion:" (map (take 16) posLines) of
```

Comment:
Uff - ok. I thought "It must be a valid YAML file." in the schema was sufficient to prevent that case.

Reply:
So, do you agree to ignore that problem for now? Or would like to take a stab at it? It's not easy to fix, I believe, as it would almost amount to writing our own JSON/YAML parser... I think I'll vote for ignoring and trusting that people will just follow the line-based YAML-layout that we use everywhere.

Reply:
Ok - let's ignore it for now. I'll add a comment for future reference.